Modelica/Thermal/FluidHeatFlow.mo

```diff
@@ -2692,7 +2691,7 @@ leads to neglect of temperature transient cv*m*der(T).
         dp = 0;
         flange_a.tau = 0;
       else
-        dp = -dp1*(1-noEvent(abs(V_flow/V_flow1)));
+        dp = -dp1*(1-V_flow/V_flow1);
         flange_a.tau*w = -dp*V_flow;
```

Comment:
Why this change? Is this related to the regression?

Reply:
No. It was a flaw since the beginning. Should I leave the flaw in?

Reply:
It would have been good to not mix this fix with the fix that should purely rectify the regression problem we are facing. Otherwise it becomes more difficult to pinpoint sources of regressions. It's probably fine to leave it in now but for the future please isolate each dedicated fix in its own commit since it can then be removed or re-applied much easier.

Reply:
@AHaumer Please remove this additional fix from this PR and file a separate PR for it, to make it show up in the Release Notes.

Since `abs(v)` expands to `noEvent (if v >= 0 then v else –v)` there is no need to have the `noEvent` operator around the `abs` function.

Reply:
see 5ce114d (my fork)